python/src/aac/validation/ValidatorPlugin.py

```diff
@@ -0,0 +1,37 @@
+"""ValidatorPlugin class."""
+from __future__ import annotations
+from attr import attrib, attrs, validators
+
+
+@attrs
+class ValidatorPlugin:
+    """
+    A class that contains all the relevant information to manage and execute validator plugins with the validation process.
+
+    Attributes:
+        name: A string with the name of the command argument
+        validation_definition: The AaC definition of the Validator Plugin
+        validation_function: The validation callback function
+        validation_arguments: Any additional arguments to supply to the validation_function
```

Comment:
Is there a need to have the `validation_` prefix, here? Since it's part of a `ValidatorPlugin`, I'd say this is not really helpful information.